css/main.css

```diff
@@ -0,0 +1,563 @@
+@import url("https://fonts.googleapis.com/css2?family=Crete+Round&family=Poppins&family=Roboto:wght@700&family=Inter:wght@400;700&display=swap");
+
+* {
+  padding: 0;
+  margin: 0;
+  box-sizing: border-box;
+}
+
+html {
+  scroll-behavior: smooth;
+}
+
+a {
+  text-decoration: none;
+}
+
+hr {
+  width: 80vw;
+}
+
+header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 15px;
+  background-color: #3c3a39;
+  color: white;
+}
+
+#desktop-nav,
+#desktop-nav ul {
+  display: none;
+}
+
+#logo {
```

Comment:
- [ ] **OPTIONAL:** It is not recommended to use IDs as CSS  selectors, I strongly recommend you to prefer classes instead of them, because classes are reusable while you can't use IDs twice or more, it will help you and saves lines in the CSS file when you want to create an element in the same style with others.
_Also for other id selectors below_